src/index.scss

```diff
@@ -129,4 +129,20 @@ p {
     font-size: 14px;
     line-height: 24px;
   }
+}
+
+.teamImg {
```

Comment:
make sure to include height and some kind of object-fit value (you can find what works best) to make sure everything is a circle instead of a oval sometimes

Reply:
instead of changing the classes based on mobile, add a conditional width attribute to your image. this way you can use just one class because it looks like there's a lot of repeat here (besides width)